charts/tigera-operator/templates/crs/custom-resources.yaml

```diff
@@ -16,6 +16,13 @@ spec:
 
 {{ end }}
 
+{{- if .Values.imageRegistry }}
```

Comment:
Per my other comment, I don't think we should add these fields. But as an aside, if we were to add them they would need to be above the `{{ end }}` clause on line 17. Otherwise, setting `values.installation.enabled` to false will result in the generation of an invalid custom resources file.

Reply:
Right, makes sense.